src/Authentication/Authentication/Properties/Resources.resx

```diff
@@ -196,4 +196,13 @@
   <data name="InferFileNameOutFilePathConflict" xml:space="preserve">
     <value>The cmdlet cannot run because the following conflicting parameters are specified: {0} and {1}. Specify either {0} or {1} then retry.</value>
   </data>
+  <data name="NotConnectedToGraphException" xml:space="preserve">
+    <value>The cmdlet cannot run because Authentication is set to {0} and Connect-Graph was not called. Invoke 'Connect-Graph' or specify Authentication to be 'UserProvidedToken' and Provide a Token then retry</value>
```

Comment:
:nit
```suggestion
    <value>The cmdlet cannot run because Authentication is set to {0} and Connect-Graph was not called. Invoke 'Connect-Graph' or specify an authentication token to 'UserProvidedToken' then retry.</value>
```

Reply:
The original version is attempting to inform the user to do the following. 
`Invoke-GraphRequest -Token $token -Authentication UserProvidedToken`